src/inverseRobot.js

```diff
@@ -24,7 +24,18 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const rob = {};
+
+  for (const r in robot) {
+    const tmp = robot[r];
```

Comment:
[VARIABLES] - use variables for the main values so that you'll be able to reuse them and give them descriptive names.